src/OpenSpaceToolkit/Physics/Environment/Objects/CelestialBodies/Earth.cpp

```diff
@@ -169,133 +81,96 @@ Earth* Earth::clone() const
     return new Earth(*this);
 }
 
-Earth Earth::Default()
+Earth Earth::GravityOnly(const Shared<EarthGravitationalModel>& aGravitatationalModel)
```

Comment:
```suggestion
Earth Earth::GravitationalOnly(const Shared<EarthGravitationalModel>& aGravitatationalModel)
```
For naming consistency?